src/Illuminate/Redis/composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^7.2",
+        "ext-redis": "*",
```

Comment:
If we move this to require then does a RedisManager class even make sense? Maybe someone still wants to use Predis or another homebrew connector and then they might not need the C implementation.

Reply:
A manaer still makes sense because of multiple redis connections.

Reply:
Yeah but my point is mainly that this makes ext-redis required but doesn't reasons with the fact that people might run other implementations.

Reply:
Hmmm, though we used to make the predis package required?

Reply:
Yeah but predis is a composer package which is generally more easily to install than a c extension.

Reply:
Right. Hopefully, in the future, FFI + preloading will actually overcome this, essentially allowing extensions to be loaded with composer! We are a bit away from this though, since it would, currently, require php apps to have their own fpm process, specially setup to load the apps own preload file.

Reply:
Alright. Moved back to suggest.